root/components/custom-navbar.js

```diff
@@ -8,18 +8,19 @@ class Navbar extends HTMLElement {
     // The shadow root will help us keep everything separated
     this.attachShadow({ mode: 'open' });
 
-    // create styles for navbar
+    // Create styles for navbar
     const style = document.createElement('style');
     style.innerHTML = `
         .navbar-container {
             background-color: #C2FFD9;
-            position: relative;
+            position: fixed;
```

Comment:
I like this a lot more for the Navbar, looks way cleaner! Curious if we should move the `<custom-searchbar>` out of this midpart div and into the `<custom-navbar>`. That way our search bar is always at the top w/ the navigation bar (makes sense logically), and I think it might even look cleaner than spank in the middle of the page.